Selenium.Axe/Selenium.Axe/AxeRunOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Selenium.Axe
+{
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class RunOnlyOptions
+    {
+        /// <summary>
```

Comment:
Was the intent to use these doc comments on all public classes/properties? It's applied inconsistently right now

Reply:
i prefer having docs for the public apis. & i try to avoid comments for internal members unless i feel it is not clear.

---
In reply to: [311814608](https://github.com/devaradhanm/SeleniumAxeDotnet/pull/1#discussion_r311814608) [](ancestors = 311814608)